src/test/kotlin/ua/marchenko/artauction/artwork/controller/ArtworkControllerTest.kt

```diff
@@ -9,24 +9,33 @@ import ua.marchenko.artauction.artwork.service.ArtworkService
 import artwork.random
 import getRandomObjectId
 import kotlin.test.Test
-import org.mockito.Mockito.mock
 import getRandomString
-import org.mockito.kotlin.doReturn
-import org.mockito.kotlin.doThrow
-import org.mockito.kotlin.whenever
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import org.junit.jupiter.api.BeforeEach
 import ua.marchenko.artauction.artwork.controller.dto.CreateArtworkRequest
 import ua.marchenko.artauction.artwork.model.Artwork
 
 class ArtworkControllerTest {
 
-    private val mockArtworkService: ArtworkService = mock(ArtworkService::class.java)
-    private val artworkController: ArtworkController = ArtworkController(mockArtworkService)
+    @MockK
+    private lateinit var mockArtworkService: ArtworkService
+
+    @InjectMockKs
+    private lateinit var artworkController: ArtworkController
+
+    @BeforeEach
+    fun setUp() {
+        MockKAnnotations.init(this)
+    }
```

Comment:
there should be way to enable annotations by default
maybe this property would help you
systemProperty("junit.jupiter.extensions.autodetection.enabled", "true")

https://junit.org/junit5/docs/current/user-guide/#extensions-registration-automatic